package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
What is the file for?

Reply:
It seems like be generated by running a `npm` command in the root diretory by mistake. It can be deleted.